findContact/FindContactModal.js

```diff
@@ -131,10 +149,59 @@ class FindContactModal extends React.Component {
     );
   }
 
+  addContacts = () => {
+    const contactList = Object.values(pickBy(this.state.checkedContactsMap));
+
+    this.props.save(contactList);
+  }
+
+  toggleItem = (contact) => {
+    const { id } = contact;
+
+    this.setState(({ checkedContactsMap }) => {
+      const newContact = checkedContactsMap[id] ? null : contact;
+
+      return {
+        checkedContactsMap: {
+          ...checkedContactsMap,
+          [id]: newContact,
+        },
+        isAllChecked: false,
+      };
+    });
+  }
+
+  toggleAll = () => {
+    const contacts = get(this.props.resources, 'records.records', []);
+
+    this.setState((state) => {
+      const isAllChecked = !state.isAllChecked;
+      const checkedContactsMap = reduceContactsToMap(contacts, isAllChecked);
+
+      return {
+        checkedContactsMap,
+        isAllChecked,
+      };
+    });
+  }
+
   render() {
     const { resources, mutator, stripes, renderNewContactBtn } = this.props;
-
+    const { checkedContactsMap, isAllChecked } = this.state;
+    const checkedContactsListLength = Object.values(pickBy(checkedContactsMap)).length;
+    const columnMapping = {
+      isChecked: <Checkbox type="checkbox" checked={isAllChecked} onChange={this.toggleAll} />,
```

Comment:
it's hard to read. I think it's a project code style to break multiple props on multiple lines

Reply:
Fixed